sponsor-dapp-v2/src/components/Step1.js

```diff
@@ -9,23 +9,29 @@ class Step1 extends Component {
     super(props);
 
     this.state = {
-      allowedToProceed: false
+      allowedToProceed: false,
+      selectedIdentifier: null
```

Comment:
What is `selectedIdentifier` used for?

Reply:
Nice catch - that was from a previous iteration and I forgot to remove it.